tests/async_fixtures/test_async_fixtures_with_finalizer.py

```diff
@@ -33,7 +34,11 @@ async def port_afinalizer():
             # RuntimeError is raised if task is created on a different loop
             await finalizer
 
-        asyncio.get_event_loop().run_until_complete(port_afinalizer())
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", DeprecationWarning)
+            loop = asyncio.get_event_loop()
+
+        loop.run_until_complete(port_afinalizer())
```

Comment:
I'm not sure about this change.

In all Python versions (at least all the ones I had interpreters installed for), this `get_event_loop()` is implicitly creating an event loop. An event loop never already exists. If you replace this with `get_running_loop()`, it always raises.

I'm not really following what this test is trying to do (what does "port" mean in this context?), but from the comment on line 33, the implicit loop creation doesn't sound intentional.

Reply:
Done, TY.